client/snapctl.go

```diff
@@ -31,6 +33,18 @@ type SnapCtlOptions struct {
 	// which context and handler should be used, etc.)
 	ContextID string `json:"context-id"`
 
+	// stdin dta read for the command (will only be set for some comamnds)
+	//
+	// TODO: Stdin as bytes should go away, instead the client
+	//       should open a connection (or re-using the exisitng
+	//       POST) for the real stdin and forward that to the
+	//       daemon. There are some complications with that, i.e.
+	//       that net.http wants to read the body of a POST and
+	//       will hang for stdin. A workaround is to use the transport
+	//       directly and to POST. Some PoC code for this:
+	//       https://github.com/snapcore/snapd/compare/master...mvo5:snapctl-real-stdin-forwarding-mess?expand=1
+	StdinData []byte `json:"stdin-data"`
```

Comment:
This can't be in SnapCtlOptions, needs to be a separate struct or something. Also should be called "stdin".